src/doc/book/if-let.md

```diff
@@ -73,8 +73,9 @@ loop {
 Into code like this:
 
 ```rust
-# let option: Option<i32> = None;
-while let Some(x) = option {
+let v: vec![1, 3, 5, 7, 11, ];
+let mut iter: v.iter();
```

Comment:
should be =, not : on these two lines.

But, why not show off something else than iterators since that's a good while let speciality? For example looping over the result of `v.pop()`.